.travis.yml

```diff
@@ -3,12 +3,42 @@ language: node_js
 node_js:
  - "8.9"
 
+services:
+  - postgresql
+
 jobs:
   include:
     - stage: test-app-waf
       install:
         - npm --prefix app/waf install
+      before_script:
+        - export PG_HOST=localhost
       script:
         - npm --prefix app/waf test
         - npm --prefix app/waf run coverage
         - npm --prefix app/waf run coveralls
+      deploy:
+        provider: script
+        on:
+          branch: master
+        script:
+          - npm --prefix app/waf start >/dev/null 2>&1 &
+          - sleep 10
+          - curl -s -X GET http://localhost:3000/adcaas/v1/ping
+          - git clone https://github.com/F5Networks/f5-adcaas-openstack.git
+          - cd f5-adcaas-openstack
+          - git checkout gh-pages
+          - HOST="localhost:3000"
+          - curl -s -X GET http://${HOST}/openapi.json > openapi.json
+          - curl -s -X GET http://${HOST}/explorer/ > index.html
+          - |
+            for file in swagger-ui.css swagger-ui-bundle.js swagger-ui-standalone-preset.js favicon-16x16.png favicon-32x32.png ; do
+              curl -s -X GET http://${HOST}/explorer/${file} > ${file}
+            done
+          - sed -i 's/LoopBack API Explorer/F5 ADCaaS API Specification/g' index.html
+          - sed -i 's/\/openapi.json/.\/openapi.json/g' index.html
+          - git config --global user.email "q.zhao@f5.com"
+          - git config --global user.name "Qin Zhao"
+          - git config --global credential.helper "$(pwd)/git-credential.sh"
+          - git commit -a -m "Travis CI $TRAVIS_BUILD_WEB_URL"
+          - git push origin gh-pages
```

Comment:
Does it use your personal own git pages?

Reply:
No. It is the github pages of F5 account. I need to make a commit message here.